backend/portal/api/portal-api.yml

```diff
@@ -418,6 +418,54 @@ paths:
         "400":
           $ref: "#/components/responses/400"
 
+  /v1/user-collections/index:
+    get:
+      tags:
+        - collections
+      summary: >-
+        Returns all public, non tombstoned collections. This endpoint is meant to be used for displaying collections
+        in a compact view. It doesn't return all the fields, so it is not suitable for general usage.
```

Comment:
This summary is copied/pasted from the above route. Let's make it accurate.

Reply:
Also, the summary should be a short sentence. Further detail should be provider in a `description:` block, as is being done with the datasets endpoint you added.

Reply:
@NoopDog move detailed description to `description` block

Reply:
Updated summary to move detail to description.